server/sonar-server/src/main/java/org/sonar/server/es/BaseIndexer.java

```diff
@@ -98,7 +100,9 @@ public BaseIndexer setEnabled(boolean b) {
 
   @Override
   public void start() {
-    lastUpdatedAt = esClient.getMaxFieldValue(indexName, typeName, dateFieldName);
+    if (!settings.getBoolean("sonar.internal.es.disableIndexes")) {
```

Comment:
`enabled` is not enough for me, I need to disable what's done in the start method too